queries.sql

```diff
@@ -8,3 +8,35 @@ SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered = 't';
 SELECT * from animals WHERE name <> 'Gabumon';
 SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+SELECT * from animals;
+ROLLBACK;
+SELECT * from animals;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+SELECT * from animals;
+
+BEGIN;
+DELETE FROM animals
+WHERE date_of_birth > 'January 1, 2022';
```

Comment:
Consider using the same date format as in the animals table.

Reply:
Hi @muneebulrehman,

Thank you for your suggestion. I think this is optional, if not can you please provide a reference to this. I have written it like this because it is easier to read.

Reply:
Hi @NayanKaran, this was optional, I just forgot to mention it. But it is important and also industry standard to be consistent with the dates.

Reply:
Hi @muneebulrehman,
"But it is important and also industry standard to be consistent with the dates." - it would help me a lot if provide a reference to this statement.

Reply:
Suppose you are using `DD/MM/YYYY`, then you should stick to this format when you are making queries. Here are using a different format compared to what you have used elsewhere. ⬇️
![image](https://user-images.githubusercontent.com/33268927/183904327-f729397f-c154-42ff-a02c-a9b163be5081.png)
Now, it does not have any measurable impact on DB but as I said, it is better to be consistent with the date format you are using. I hope I am clear with this.